src/test/app/CrossingLimits_test.cpp

```diff
@@ -558,8 +558,11 @@ class CrossingLimits_test : public beast::unit_test::suite
         using namespace jtx;
         auto const sa = supported_amendments();
         testAll(sa);
-        testAll(sa - featureFlowSortStrands);
-        testAll(sa - featureFlowCross - featureFlowSortStrands);
+        testAll(sa - featurePermissionedDEX);
+        testAll(sa - featureFlowSortStrands - featurePermissionedDEX);
```

Comment:
I don't think we need to test these two without `PermissionedDEX` since these amendments are enabled. A similar comment also applies to quite a few other added unit-tests in other modules.